src/components/Calculator.js

```diff
@@ -0,0 +1,37 @@
+function Calculator() {
```

Comment:
- [ ] Please, Calculator should be a class-based component, right now is a function component. I kindly invite you to refactor it. In the next steps, you'll see why this is important for the project. 🌵

Reply:
Thanks, Vichuge for the point that out. I will make the changes right away.